UI/css/page.css

```diff
@@ -0,0 +1,81 @@
+@import url(https://fonts.googleapis.com/css?family=Raleway:400,700);
+
+body{
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  background: #c0392b;
+  font-family: Raleway;
+  color: white;
+  letter-spacing: 1px;
+}
+
+.card-form{
+  width: 500px;
+  border-radius: 10px;
+  background: white;
+  box-shadow: 0 27px 55px 0 rgba(0, 0, 0, 0.3), 0 17px 17px 0 rgba(0, 0, 0, 0.15);
+}
+  
+.form-title{
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 80px;
+    font-size: 20px;
+    font-weight: bold;
+    background: #27ae60;
+    border-radius: 10px 10px 0 0;
+}
+  
+.form-body{
+    padding: 10px;
+}
+    
+.row{
+      display: flex;
+      justify-content: space-around;
+      padding: 10px;
+}
+      
+   input[type="text"]{
+        appearance: none;
+        width: 100%;
+        margin: 0 15px;
+        height: 35px;
+        padding: 5px 15px;
+        border-radius: 5px;
+        outline: none;
+        border: none;
+        background: #e8ebed;
+        color: #576366;
+        font-size: 14px;
+}
+
+.rule{
+    height: 2px;
+    background: #e8ebed;
+    margin: 0px 35px;
+}
+  
+.form-footer{
+    margin: 0 25px 15px 25px;
+    padding: 15px 10px;
+}
+     a{
+      display: inline-block;
+      height: 40px;
+      line-height: 40px;
+      border: none;
+      border-radius: 5px;
+      padding: 5px 15px;
+      background: #27ae60;
+      margin-right: 10px;
+      font-size: 14px;
+    }
+    
+span{
+      margin-left: 8px;
+    }

```

Comment:
Why is this in a folder called UI instead of being in server/public?

Reply:
Also, why did you include a bunch of dog pictures in the project?

Reply:
page2.css and index.css appear to have completely separate custom css files. This is not going to work. Instead, you should have made a single uniform layout used by all pages.

Reply:
Also, why are we using 6 different fonts?